employee-crud/src/utils/helper.js

```diff
@@ -0,0 +1,32 @@
+import dotenv from "dotenv";
+import path from "path";
+import * as fsPromise from "fs/promises";
+import * as fs from "fs";
+
+dotenv.config();
+
+const dbPath = path.join(path.resolve(), process.env.DB_PATH);
+
+export const getDbData = async () => {
+  if (!fs.existsSync(dbPath)) {
+    await fsPromise.writeFile(dbPath, "[]", { encoding: "utf-8" });
+  }
+  let dbData = await fsPromise.readFile(dbPath, { encoding: "utf-8" });
+  return (dbData = JSON.parse(dbData));
+};
+
+export const generateId = async () => {
+  const dbData = await getDbData();
+  if (dbData.length === 0) return 1000;
```

Comment:
why you are returning 1000. can you please explain?

Reply:
If there are no data in db file then it should start with 1000.

Reply:
ok